### PR TITLE
Add GMT+2 timezone

### PR DESCRIPTION
**Pull request by cfanoulis**

### CODE REVIEW

File: make-node-meeting.sh [hunk 1]
```diff
@@ -69,12 +69,14 @@ Node.js Foundation $GROUP_NAME Meeting $utc_short
...
 | London        | $(TZ=Europe/London date --date="$meeting_date" +"$common_fmt") |
 | Amsterdam     | $(TZ=Europe/Amsterdam date --date="$meeting_date" +"$common_fmt") |
+| Athens        | $(TZ=Europe/Athens date --date="$meeting_date" + "$common_fmt") |
 | Moscow        | $(TZ=Europe/Moscow date --date="$meeting_date" +"$common_fmt") |
 | Chennai       | $(TZ=Asia/Kolkata date --date="$meeting_date" +"$common_fmt") |
 | Hangzhou      | $(TZ=Asia/Shanghai date --date="$meeting_date" +"$common_fmt") |
 | Tokyo         | $(TZ=Asia/Tokyo date --date="$meeting_date" +"$common_fmt") |
 | Sydney        | $(TZ=Australia/Sydney date --date="$meeting_date" +"$common_fmt") |
 
+
```
Comment (Trott) [inline make-node-meeting.sh]: Unrelated whitespace change?

Comment (cfanoulis) [reply make-node-meeting.sh]: Yup, blame VSC. Fixing

Comment (cfanoulis) [reply make-node-meeting.sh]: Marking as resolved with https://github.com/nodejs/make-node-meeting/pull/36/commits/000c0ee2adb3a99fd66316143c2c1bb061ee2d3b
